tests/e2e/auth-api.spec.ts

```
@@ -0,0 +1,59 @@
+import { expect, test } from '@playwright/test'
+
+test.describe('Authentication API Tests', () => {
+  test('should authenticate via API directly', async ({ request, context }) => {
```

ESLint

[error] 'context' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)